src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -28,5 +30,26 @@
     [quo/network-bridge
      {:amount          amount
       :network         to-network
-      :status          :default
+      :status          status
       :container-style {:right 12}}]]])
+
+(defn view
+  [{:keys [amount route networks input-value]}]
+  (let [loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
+        from-network              (utils/id->network (get-in route [:From :chainId]))
+        to-network                (utils/id->network (get-in route [:To :chainId]))]
+    [rn/scroll-view
+     {:content-container-style {:flex-grow       1
+                                :align-items     :center
+                                :justify-content :center}}
+     (when (not (empty? input-value))
+       (if (and (not loading-suggested-routes?) route)
+         [routes
+          {:amount       amount
+           :status       :default
+           :from-network from-network
+           :to-network   to-network}]
+         [routes
+          {:status       :loading
+           :from-network (:network-name (nth (seq networks) 1))
+           :to-network   (:network-name (nth (seq networks) 1))}]))]))
```

Comment:
Maybe we can simplify this code by having one routes element and set the corresponding value of `from-network`, `to-network`, `status` and `amount` in the let bindings